src/remoteConfig.ts

```diff
@@ -10,7 +9,6 @@ type RemoteConfigValue = {
   vordrIps: string[];
   blockedCountries: string[];
   ignoredWorkEmailDomains: string[];
-  pricingIds: Record<string, SubscriptionCycles>;
```

Comment:
Webhook is no longer dependent on the defined pricing IDs from the remote config. Everything should function accordingly without needing to update anything.